test/CAS/plugin_cas.swift

```diff
@@ -74,6 +74,30 @@ import SubE
 // CHECK-DAG:     "swift": "A"
 // CHECK-DAG:     "swift": "F"
 
+/// --------Swift module F
```

Comment:
Is this order change intentional?

Reply:
I moved around the ordering back to what it was prior to this change, now.